test/mailbox.js

```diff
@@ -45,7 +45,11 @@ module.exports = function (host, port, printLogs) {
         log('mail body', body)
         var json = null
         try {
-          json = JSON.parse(body)[0]
+          json = JSON.parse(body)
+
+          if (json.length === 1) {
```

Comment:
Updated this to return an array of emails (if more than one) or a single email object if only one (backwards compat).